main.yml

```diff
@@ -352,8 +350,17 @@ settings:
   title: Cloud Services Settings
   frontend:
     sub_apps:
+      - id: ''
+        default: true
+        title: 'User Preferences'
       - id: hooks
+      - id: sources
+      - id: rbac
   top_level: true
+  isBeta: true
+  deployment_repo: https://github.com/RedHatInsights/settings-frontend-build
+  paths:
+  - /settings
```

Comment:
There's going to be another path in newar future for settings app `/settings/app` I don't really know how to structure the ID here @kruai any ideas. The url has to be `/settings/${smething}` and the `something` should indicate that it's a app settings. ID with `app` sounds strange and someone reading this file could understand it as placeholder.